using_images/s2i_images/python.adoc

```diff
@@ -33,32 +33,50 @@ https://github.com/openshift/sti-python/tree/master/3.5[3.5] of Python.
 
 == Images
 
+ifdef::openshift-online[]
+RHEL 7 images are available through Red Hat's subscription registry:
```

Comment:
s/Red Hat's subscription registry/the Red Hat Registry/

See https://access.redhat.com/containers/#/faq.